docusaurus/docs/shared/plugin-anatomy.md

```diff
@@ -12,12 +12,12 @@ While certain plugin types can have specific configuration options, let's look a
 
 To see all the available configuration settings for the `plugin.json`, refer to the [plugin.json Schema](../metadata.md).
 
-### `module.ts`
+### `module.js`
```

Comment:
I think `ts` in the title is better as we're talking about the two files that you interact with as a developer and we name these after the h2. The other changes below make sense

Reply:
Fair enough. I've reverted the title. 👍